apps/stops/lib/stop.ex

```diff
@@ -190,6 +194,17 @@ defmodule Stops.Stop.ParkingLot.Capacity do
   defp pretty_parking_type(0), do: nil
   defp pretty_parking_type(1), do: "Garage"
   defp pretty_parking_type(2), do: "Surface Lot"
+
+  @spec pretty_overnight_msg(String.t() | nil) :: String.t()
+  defp pretty_overnight_msg("no"), do: "Not available"
+  defp pretty_overnight_msg("yes"), do: "Available"
+  defp pretty_overnight_msg("yes-except-snow"), do: "Available, except during snow emergencies"
+  defp pretty_overnight_msg("no-except-snow"), do: "Not available, except during snow emergencies"
+
+  defp pretty_overnight_msg("yes-snow-unknown"),
+    do: "Available. During snow emergencies, check posted signs."
+
+  defp pretty_overnight_msg(_), do: "Unknown"
```

Comment:
Oh, hm, just seeing that we do this now, so we'd always have a value for overnight parking. In that case you can change the interface to remove the ? instead of my comment above.